default-templates/server/favicon.ico.acl

```diff
@@ -0,0 +1,15 @@
+# ACL for the default favicon.ico resource
+# Server operators will be able to override it as they wish
```

Comment:
I don't see any Control option being set though.

Reply:
Ok, so, that’s intentional. For the server templates, we don’t currently have any notion of giving identity to the server operator (like an admin user or whatever). So it’s unclear who to give write/control permissions to. 
More importantly, the assumption is that server operators will be able to edit this acl file directly. (via the file system)